### DISCUSSION TITLE
如何使用 alist-aria2 容器中预装的 aria2？

### DISCUSSION THREAD

**Discussion by LAST7** [AlistGo/alist#5886] [Q&A]
要如何使用 alist-aria2 中预装的 aria2？我启动了这个镜像的容器，但是 6800 端口提示拒绝连接：

```
Failed get aria2 version: Post "http://localhost:6800/jsonrpc": dial tcp *********:6800: connect: connection refused
```

进容器 netstat 了一下发现没有程序在监听 6800 端口，但是 ps -a 可以看到一个 [aria2c] 的进程，颇为离奇

**Comment by pansila** [comment]
https://github.com/alist-org/alist/issues/5943
